src/index.js

```diff
@@ -17,12 +17,13 @@ const merge = require('lodash/merge');
 const path = require('path');
 const Chalk = require('chalk').constructor;
 const { diffImageToSnapshot } = require('./diff-snapshot');
+const fs = require('fs');
 
 function updateSnapshotState(oldSnapshotState, newSnapshotState) {
   return merge({}, oldSnapshotState, newSnapshotState);
 }
 
-function toMatchImageSnapshot(received, { customSnapshotIdentifier = '', customDiffConfig = {}, noColors = false } = {}) {
+function toMatchImageSnapshot(received, { customSnapshotIdentifier = '', customDiffConfig = {}, noColors = false, cleanPassingDiffs = true } = {}) {
```

Comment:
Can we remove `cleanPassingDiffs` as an option and just always clean up those diffs? If there is no diff `toMatchImageSnapshot` does not even tell the user that there is an output file available, it just passes, and I can't think of any reason why you would want to see the diff file when it would be blank.